### ISSUE TITLE
nodejs is not defined

### ISSUE THREAD

**Issue by thepragmaticdev** [janeasystems/nodejs-mobile#284]
* **Project**: nodejs-mobile-cordova
* **Version**: 0.4.1
* **Mobile device**: Android Studio Emulator
* **Mobile OS and version**: Android 10
* **Development Node.js**: 12.11.0
* **Development platform**: Windows 10 64-bit

Logcat:
2020-10-03 20:52:36.330 4501-4501/io.cordova.hellocordova I/chromium: [INFO:CONSOLE(49)] "Uncaught ReferenceError: nodejs is not defined", source: file:///android_asset/www/js/index.js (49)

**Comment by thepragmaticdev** [comment]
@jaimecbernardo I will donate you $100 if you solve this.

**Comment by asaadbaw** [comment]
Does anyone have a solution?

**Comment by asaadbaw** [comment]
@thepragmaticdev did you find a solution?

**Comment by thepragmaticdev** [comment]
@asaadbaw No, but what I've found is that this: https://github.com/JaneaSystems/nodejs-mobile-cordova/tree/unstable/testapp works.

**Comment by jaimecbernardo** [comment]
Most of the times that this happens is when there's something missing in the development environment.
